token/rust/src/token.rs

```diff
@@ -299,14 +315,14 @@ where
 
     /// Create and initialize the associated account.
     pub async fn create_associated_token_account(&self, owner: &Pubkey) -> TokenResult<Pubkey> {
-        self.process_ixs(
+        self.process_ixs::<[&dyn Signer; 0]>(
```

Comment:
Very slick, I had no idea how to resolve this one

Reply:
It's a little funny and mind-bending to have to define the type of emptiness 😆